kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KieServerMigrationTest.java

```diff
@@ -122,8 +126,12 @@ public void setup() {
     
     @After
     public void cleanup() {
-        kieServicesClient.disposeContainer(containerId);
-        kieServicesClient.disposeContainer(containerId2); 
+        if (kieServicesClient != null) {
+            ServiceResponse<Void> response = kieServicesClient.disposeContainer(containerId);
+            logger.info("Container {} disposed with response - {}", containerId, response.getMsg());
+            kieServicesClient.disposeContainer(containerId2);
+            logger.info("Container {} disposed with response - {}", containerId, response.getMsg());
```

Comment:
wrong trace, it should be bind to containerId2
```suggestion
            logger.debug("Container {} disposed with response - {}", containerId2, response.getMsg());
```

Reply:
thanks a lot, I missed that!